framework/test/mocha/unit/modules/chain/submodules/blocks/verify.js

```diff
@@ -1794,8 +1794,8 @@ describe('blocks/verify', () => {
 					});
 
 					it('should throw error when transaction is already confirmed', async () => {
-						expect(__private.checkTransactions(dummyBlock, true)).to.eventually
-							.rejected;
+						return expect(__private.checkTransactions(dummyBlock, true)).to
+							.eventually.rejected;
```

Comment:
Would be nice to be explicit here and add the expected message using `rejectedWith` instead of `rejected` only